src/v1/providers/firestore.ts

```diff
@@ -31,9 +31,12 @@ import * as logger from "../../logger";
 import { CloudFunction, Event, EventContext, makeCloudFunction } from "../cloud-functions";
 import { DeploymentOptions } from "../function-configuration";
 
-export const provider = "google.firestore";
-export const service = "firestore.googleapis.com";
-export const defaultDatabase = "(default)";
+/** @internal */
+export const PROVIDER = "google.firestore";
```

Comment:
"technically" renaming an expose variable name is a breaking change.

Truthfully, I don't really care for that kind of breaking changes, but I'd err on the safe side and not change the casing here unless there are other practical purpose I'm missing.

Reply:
Fair point, reverted this. 👍